docs/source/basic-usage.rst

```diff
@@ -98,21 +100,23 @@ threading, greenlets, asyncio, etc.
 
 You also need to provide the application glue. To send a WebSocket message::
 
-    ws.send_data("Hello world!")
+    ws.send(Data(data="Hello world!"))
```

Comment:
Is specifying the `data` keyword arg really necessary? In the spirit of KISS, I would omit it in all examples and docs.

Reply:
One more comment: this creates a `Data` object, but the documentation states `... to send a WebSocket message ...`. Should we consider renaming it to plain, e.g., `Message(data='foobar')`?

Reply:
Reading a bit further, is this a mixup with the handshake "data"? Because there is now the RejectData event... which is badly named if I understand it correctly, that it applies only to the handshake?

Reply:
`Data` is renamed `Message` in a commit I've just pushed.